libs/blocks/marketo/marketo.css

```diff
@@ -533,7 +533,7 @@
   background-color: #123456
 }
 
-.resource-form .marketo {
+.resource-form.section.two-up .marketo {
```

Comment:
Does this need to work with other ups? You could do `.resource-form.section[class*="-up"] .marketo`

Reply:
The marketo block is too large for other ups, I'm not sure that makes sense without new designs.